velox/exec/tests/TaskTest.cpp

```diff
@@ -627,6 +628,55 @@ TEST_F(TaskTest, supportsSingleThreadedExecution) {
   ASSERT_FALSE(task->supportsSingleThreadedExecution());
 }
 
+TEST_F(TaskTest, updateBroadCastOutputBuffers) {
+  auto plan = PlanBuilder()
+                  .tableScan(ROW({"c0"}, {BIGINT()}))
+                  .project({"c0 % 10"})
+                  .partitionedOutputBroadcast({})
+                  .planFragment();
+  auto bufferManager = PartitionedOutputBufferManager::getInstance();
+  // Test for SUCCESS and NO_OP cases.
+  {
+    auto taskId = "single.execution.task.0";
+    auto task = std::make_shared<exec::Task>(
+        taskId,
+        plan,
+        0,
+        std::make_shared<core::QueryCtx>(driverExecutor_.get()));
+    {
+      auto bufferManagerShared = bufferManager.lock();
+      bufferManagerShared->initializeTask(task, true, 2, 2);
+    }
+    ASSERT_EQ(
```

Comment:
Use ASSERT_TRUE and ASSERT_FALSE. Update comments to remove references to SUCCESS, NO_OP and BUFFERS_NOT_FOUND enum values that no longer exist.